app/views/application/_radius_details.html.erb

```diff
@@ -7,8 +7,12 @@
   <div class="govuk-details__text">
     <p> Your RADIUS secrets are:</p>
```

Comment:
I'd sink lines 8 and 9 into the `if` block - don't need the header or the list unless there are actually secret keys.